packages/twenty-front/src/modules/object-record/record-group/types/RecordGroupDefinition.ts

```diff
@@ -0,0 +1,30 @@
+import { ThemeColor } from 'twenty-ui';
+
+export const enum RecordGroupDefinitionType {
+  Value = 'value',
+  NoValue = 'no-value',
+}
+
+export type RecordGroupDefinitionNoValue = {
```

Comment:
It can be, I just use the same type that was used before and renamed it.
We can create a const and infer the type, but I think I rather prefer this approach for now I found it more clear.